### PR TITLE
accept section number and subpages

### PR DESCRIPTION
**Pull request by Konfekt**
Closes https://github.com/lambdalisue/vim-manpager/issues/8

### CODE REVIEW

File: plugin/manpager.vim [hunk 1]
```diff
@@ -9,9 +9,8 @@ function! s:MANPAGER() abort
   call manpager#manpagerlize()
 endfunction
 function! s:MAN(...) abort
-  let sect = get(a:000, 0, '')
-  let page = get(a:000, 1, sect)
-  let sect = sect ==# page ? '' : sect
+  let sect = a:1 =~ '\v\d+' ? a:1 : ''
+  let page = join(a:000[ (empty(sect) ? 0 : 1) :], '-')
```
Comment (lambdalisue) [inline plugin/manpager.vim]: What happen if `a:000` is empty (`call s:MAN()`)?

**Comment by Konfekt** [comment]
Ok, allow 0 arguments and look up cword in that case.

**Comment by lambdalisue** [comment]
LGTM :-)

**Comment by lambdalisue** [comment]
Thanks :-)

**Pull request merged**